platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -578,6 +581,18 @@ class EditorEditText : AppCompatEditText {
         )
     }
 
+    override fun addTextChangedListener(watcher: TextWatcher) {
+        if (!this::textWatcher.isInitialized) {
+            this.textWatcher = EditorTextWatcher()
+            super.addTextChangedListener(this.textWatcher)
+        }
+        textWatcher.addChild(watcher)
+    }
+
+    override fun removeTextChangedListener(watcher: TextWatcher) {
+        textWatcher.removeChild(watcher)
```

Comment:
If there is no child, remove the textWatcher from parent too?

Reply:
I don't think it's worth it, to be honest. Keeping it there has no cost as far as I can tell, and will also be needed to notify the composer of any changes made by any external actors as the spell checker or some suggestion replacements.